webapp/channels/src/sass/routes/_get-app.scss

```diff
@@ -138,26 +139,11 @@
     }
 }
 
-.get-app__checkbox {
+input[type="checkbox"].get-app__checkbox {
     width: 18px;
     height: 18px;
     padding: 0;
```

Comment:
can you nest this inside of `get-app__preference` that way we dont have to use this selector for specificity `input[type="checkbox"].`